### PR TITLE
add documentation radiation parameters

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request closes some aspects of issue #1982 by documenting in detail the parameters of the radiation plugin.

It will **not** update white spaces.

### CODE REVIEW

File: src/picongpu/include/simulation_defines/param/radiation.param [hunk 1]
```diff
@@ -21,11 +21,21 @@
...
+ * Definition of frequency space, number of observers, filters, form factors and
+ * window functions of the radiation plugin.
+ *
+ * All values set here determine what the radiation plugin will compute.
+ * The observation direction is defined in a seperate file `radiationObserver.param`.
+ * On the comand line the plugin still needs to be called for each species the radiation
+ * should be computed for.
+ */
 
+  /**
```
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/radiation.param]: this docstring has wrong alignment :)

File: src/picongpu/include/simulation_defines/param/radiation.param [hunk 2]
```diff
@@ -21,11 +21,21 @@
 
 #pragma once
```
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/radiation.param]: while we are on it, please write this order and number of empty newlines in the header:
```C++
/* Copyright ...
 */

/** @file
 *
 * ...
 *
 * ...
 */

#pragma once


// start code ...
```

File: src/picongpu/include/simulation_defines/param/radiationObserver.param [hunk 3]
```diff
@@ -20,6 +20,14 @@
 
 #pragma once
 
+/** @file
```
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/radiationObserver.param]: same here as in `radiation.param`

**Comment by PrometheusPi** [comment]
@ax3l fixed the positional mistake of the doxygen comments

**Pull request merged**